hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/DBStoreBuilder.java

```diff
@@ -142,10 +134,6 @@ private DBStoreBuilder(ConfigurationSource configuration,
     this.rocksDbStat = configuration.getTrimmed(
         OZONE_METADATA_STORE_ROCKSDB_STATISTICS,
         OZONE_METADATA_STORE_ROCKSDB_STATISTICS_DEFAULT);
-    this.rocksDbCfWriteBufferSize = (long) configuration.getStorageSize(
```

Comment:
Since OZONE_METADATA_STORE_ROCKSDB_CF_WRITE_BUFFER_SIZE is no longer used, the TestOMSnapshotDAG should also be rewritten. It is still using it.
In addition, I think we need a default ini file to set `write_buffer_size` as the default size in Ozone is different from rocksdb.

Reply:
Most other configs are few enough that they are hardcoded programatically into Ozone, I'm not sure we want to change that here for the default ini. However having an example ini would be more failsafe when the user gets control.

I'm not sure how the default ini should be handled for example in test environments. If we rely on the default ini being always there then I'm not sure how to simply do it for all test environments. Do you have a recommendation? In the case we have to modify most docker-yaml files I'd prefer to defer that change to a separate Jira.